### PR TITLE
IFC Alginment export

### PR DESCRIPTION
**Pull request by IBuT-Software**
**ATTENTION:** This template provides structure for 
- a specific pull request in scope of IFC4.3 validation work within IFC Infrastructure Extension Deployment project.
Please fill in the corresponding section(s) and delete the rest.

@pjanck

More information in the readme.md

### CODE REVIEW

File: IFC 4.3/Alignment_1/readme.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+### Supporting files
+
+Following files correspond to this scenario:
+
+| Filename                          | Description                               |
+|-----------------------------------|-------------------------------------------|
+| `Alignment-3D.ifc`                 | the exported content as IFC document      |
+| `Alignment-3D+OffsetCurve.ifc`     | the exported content as IFC document|
+| `Alignment-3D.jpg`                 | screenshot of the alignment-3d.ifc in DESITE md pro |
+| `Properties.jpg`                 | screenshot of the properties in DESITE md pro |
```
Comment (pjanck) [inline IFC 4.3/Alignment_1/readme.md:50]: It seems a bit odd that CantDeficiency would be 0.13km? Is there a conversion error somewhere?

**Review by pjanck — CHANGES_REQUESTED** [review]
Great to see some alignments trickling in!

I am also happy to see OffsetCurveByDistances. Perhaps a good thing would be to split this into a separate Unit test? (so having 2 separate tests, one building upon the other)

File: IFC 4.3/Alignment_1/readme.md (hunk 1 above)
Comment (IBuT-Software) [reply IFC 4.3/Alignment_1/readme.md:50]: I'll let that check..

**Comment by pjanck** [comment]
We're moving to the new repository: https://github.com/bSI-InfraRoom/IFC-infra-unit-test

Please open a new pull request there and close this one.

**Pull request closed without merging**